conans/test/integration/command_v2/list_test.py

```diff
@@ -486,24 +508,33 @@ def test_list_package_id_latest_prev(self, client, remote):
                 b58eeddfe2fd25ac3a105f72836b3360 (2023-01-10 23:13:12 UTC)
                   packages
                     ebec3dc6d7f6b907b3ada0c3d3cdc83613a2b715
-                      revisions
-                        d9b1e9044ee265092e81db7028ae10e0 (2023-01-10 23:13:12 UTC)
+                      info
```

Comment:
I might be wrong, but this new bit would need to be added to https://github.com/conan-io/docs/pull/2902/files#diff-e4d16580ee2fa6a9bcd5e58dcefaeff4e86fea60cc2d9ae2044594b5954a93c6R240 , right?

Reply:
Yes, a new PR to the docs is necessary if this moves forward as-is. Part of the reason of this PR was the #TODOs I found while writing the docs, that left me a bit concerned about a couple of scenarios not yet covered.